src/views/webview-app/connection-form.tsx

```diff
@@ -43,6 +79,20 @@ const ConnectionForm: React.FunctionComponent<{
       data-testid="connection-form-modal"
       size="large"
     >
+      {isConnecting && (
+        <div
+          className={cx(
+            connectingContainerStyles,
+            darkMode && connectingContainerDarkModeStyles
+          )}
+        >
+          <CancelLoader
+            onCancel={onCancelConnectClicked}
+            progressText="Connecting..."
```

Comment:
```suggestion
            progressText="Connecting…"
```

(feel free to ignore :))